manual/development-setup/signed-commits.md

```diff
@@ -0,0 +1,52 @@
+# Configure Signed Commits
+
+[Signed Commits](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits) are used to help to protect against supply chain & impersonation attacks by ensuring Git commits are verified as genuine through the use of GPG keys. 
+
+An alternative setup guide is [here](https://withblue.ink/2020/05/17/how-and-why-to-sign-git-commits.html).
+
+## Local Configuration
+Install a GPG Client:
+
+A GPG Client needs to be installed on your development machine. For Windows this can be [GPG4Win](https://www.gpg4win.org/). For MacOS, Use [Homebrew](https://brew.sh/): `brew install gpg`. For Linux, you probably have GPG installed out of the box, but if not GPG packages should be easily obtainable via your package manager of choice.
+
+Generate a key via the command line terminal:
+`gpg --gen-key`
+Use the options:
```

Comment:
GnuPG doesn't offer the options if you use `--gen-key` You need to use `gpg --full-generate-key` This is on Linux so it might work properly on Mac or Windows but maybe worth adding in the option.